### PR TITLE
add Mainmatter telemetry workshop May 28+29

### PR DESCRIPTION
**Pull request by marcoow**
see https://ti.to/mainmatter/rust-telemetry-may-2024

### CODE REVIEW

File: draft/2024-05-15-this-week-in-rust.md [hunk 1]
```diff
@@ -191,6 +191,8 @@ Rusty Events between 2024-05-15 - 2024-06-12 🦀
     * [**Rust Hack and Learn**](https://meet.jit.si/RustHackAndLearnBerlin) | [**Mirror: Rust Hack n Learn Meetup**](https://www.meetup.com/rust-berlin/events/298477699/)
 * 2024-05-28 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/dallasrust/)
     * [**Last Tuesday**](https://www.meetup.com/dallasrust/events/300533392/)
+* 2024-05-28 & 2024-05-28 | Virtual | [Mainmatter](https://mainmatter.com/)
+    * [**Remote Workshop: Telemetry for Rust APIs – you can't fix what you can't see**](https://ti.to/mainmatter/rust-telemetry-may-2024)
```
Comment (mariannegoldin) [inline draft/2024-05-15-this-week-in-rust.md:195]: Adding that this is a paid workshop
```suggestion
* 2024-05-28 & 2024-05-28 | Virtual | [Mainmatter](https://mainmatter.com/)
    * [**Remote Workshop: Telemetry for Rust APIs – you can't fix what you can't see (fee)**](https://ti.to/mainmatter/rust-telemetry-may-2024)
```

File: draft/2024-05-15-this-week-in-rust.md [hunk 2]
```diff
@@ -191,6 +191,8 @@ Rusty Events between 2024-05-15 - 2024-06-12 🦀
     * [**Rust Hack and Learn**](https://meet.jit.si/RustHackAndLearnBerlin) | [**Mirror: Rust Hack n Learn Meetup**](https://www.meetup.com/rust-berlin/events/298477699/)
 * 2024-05-28 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/dallasrust/)
     * [**Last Tuesday**](https://www.meetup.com/dallasrust/events/300533392/)
+* 2024-05-28 & 2024-05-28 | Virtual | [Mainmatter](https://mainmatter.com/)
```
Comment (mariannegoldin) [inline draft/2024-05-15-this-week-in-rust.md:194]: Fix date and format
```suggestion
* 2024-05-28 - 2024-05-29 | Virtual | [Mainmatter](https://mainmatter.com/)
```

Comment (mariannegoldin) [reply draft/2024-05-15-this-week-in-rust.md:194]: Fix date and format

**Review by mariannegoldin — APPROVED** [review]
Approved with formatting fix and note that this is a paid workshop.

**Comment by marcoow** [comment]
Thanks – wasn't aware paid workshops should be marked!

**Pull request merged**